src/Comment.jsx

```diff
@@ -0,0 +1,30 @@
+import { Context } from "./App";
+import { useContext } from "react";
+import Propic from "./Propic";
+
+export default function Comment({ commentText, commenterId }) {
+  const [currentUser, contacts] = useContext(Context);
+
+  let currentContact = {};
+
+  if (commenterId == 0) {
+    currentContact = currentUser;
+  } else {
+    currentContact = contacts.find((target) => target.id === commenterId);
+  }
+
+  return (
+    <div className="comment">
+      <Propic
```

Comment:
name of this compnent is not too clear, probably better to choose a clearer name, like UserInitials or something like this